gatsby-config.js

```diff
@@ -1,10 +1,21 @@
 const languages = require('./src/data/languages');
 const autoprefixer = require('autoprefixer');
 
+const ENV_PATH = `.env.${process.env.NODE_ENV}`;
+
 require("dotenv").config({
-  path: `.env.${process.env.NODE_ENV}`,
+  path: ENV_PATH,
 });
 
+if (!process.env.CONTENTFUL_SPACE_ID ||
+    !process.env.CONTENTFUL_TOKEN) {
+  console.log(
+    `Please define CONTENTFUL_SPACE_ID and ` +
+    `CONTENTFUL_TOKEN in ${ENV_PATH}.`
+  );
+  process.exit(1);
```

Comment:
The exception that's eventually thrown if these aren't defined is _super_ confusing and doesn't bring up any search results on google, so I adopted the "crash early, crash often" mantra here.